Update redirect for APD namespace (#3963)
perma-id/w3id.org: APD/README.md

This W3ID provides a persistent URI namespace for the AusTraits Plant Dictionary (APD).

# AusTraits Plant Dictionary (APD)

> The AusTraits Plant Dictionary (APD) is a vocabulary that formally defines the more than 500 plant trait concepts. It was developed in conjunction with [AusTraits](https://austraits.org), a database of Australian plant traits. 

Reviewer: ```suggestion
The AusTraits Plant Dictionary (APD) is a vocabulary that formally defines more than 500 plant trait concepts. It was developed in conjunction with [AusTraits](https://austraits.org), a database of Australian plant traits. 
```

It is expected to expand to include additional trait concepts. Each trait definition has been reviewed by multiple people and includes references and links to identical/similar traits in other trait databases whenever possible. The APD includes machine-readable formats and endpoints, allowing the traits to be readily re-used by other databases.

Suggestions for changes are welcome and can be posted at https://github.com/traitecoevo/APD/issues/.

## Vocabularies

* https://w3id.org/APD/
* https://w3id.org/APD/traits/
* https://w3id.org/APD/glossary/
* https://w3id.org/APD/releases/

## Homepage

* https://w3id.org/APD/
* https://github.com/traitecoevo/APD/
* http://austraits.org/

## Design

> - the .htaccess code is written to process calls to variants of the machine-readable representations before any queries to html, i.e. *.ttl, *.nq, *.nt, *.json before *.html. This reduces the number of RewriteCond statements needed in the html section. 

Reviewer: ```suggestion
- the `.htaccess` code is written to process calls to variants of the machine-readable representations before any queries to `html`, i.e., `*.ttl`, `*.nq`, `*.nt`, and `*.json`, before `*.html`. This reduces the number of `RewriteCond` statements needed in the `html` section. 
```

> - tested with https://htaccess.madewithlove.com (using https://w3id.org/ as the base URL, using https://w3id.org/APD/ won't work as code is written as though you're already in that subfolder)

Reviewer: ```suggestion
- tested with https://htaccess.madewithlove.com (using `https://w3id.org/` as the base URL; `https://w3id.org/APD/` won't work, as code is written as though you're already in that subfolder)
```

- the following flags are used

>   - [R=303] A redirect: when a server responds with a 303 status code, it provides a `Location` pointing to a different URI. The client, upon receiving a 303 response, automatically makes a GET request to the URI specified in the `Location` header.  
>   - [L]	If the previous conditions pass this rule is executed and no more evaluation is done. 
>   - [NE] noescape Prevent special characters being converted to their hexcode equivalent, needed for links including a `#`

Reviewer: ```suggestion
  - `[R=303]` — A redirect. When a server responds with a `303` status code, it provides a `Location` pointing to a different URI. The client, upon receiving a `303` response, automatically makes a `GET` request to the URI specified in the `Location` header.  
  - `[L]` — If the previous conditions pass, this rule is executed, and no more evaluation is done. 
  - `[NE]` — (noescape) Prevents conversion of special characters to their hexcode equivalent; needed for links including a `#`.
```
